doc/quickstart/ncWMS_quickstart.rst

```diff
@@ -1,38 +1,41 @@
 :Author: Guy Griffiths (guy.griffiths@the-iea.org)
-:Version: osgeolive7.0
+:Reviewer: Felicity Brand (Google Season of Docs 2019)
+:Version: osgeolive12.0
```

Comment:
I couldn't get it to load on my instance of version 13 but that could be due to my flaky VM.